src/common/marlin_client.c

```diff
@@ -154,24 +155,32 @@ int marlin_client_id(void) {
     return 0;
 }
 
-int marlin_client_set_dialog_open_cb(dialog_open_cb_t cb) {
+int marlin_client_set_fsm_create_cb(fsm_create_t cb) {
     marlin_client_t *client = _client_ptr();
     if (client && cb) {
-        client->dialog_open_cb = cb;
+        client->fsm_create_cb = cb;
         return 1;
     }
     return 0;
 }
 
-int marlin_client_set_dialog_close_cb(dialog_close_cb_t cb) {
+int marlin_client_set_fsm_destroy_cb(fsm_destroy_t cb) {
     marlin_client_t *client = _client_ptr();
     if (client && cb) {
-        client->dialog_close_cb = cb;
+        client->fsm_destroy_cb = cb;
         return 1;
     }
     return 0;
 }
 
+int marlin_client_set_fsm_change_cb(fsm_change_t cb) {
```

Comment:
I have no clue what cb is. Return value is not documented.

Reply:
commented